docker/Dockerfile.dmqnode

```diff
@@ -0,0 +1,26 @@
+# Duplicate definition of DIST before FROM is needed to be able to use it
+# in docker image name:
+ARG  DIST=xenial
+FROM sociomantictsunami/dlang:$DIST-v5 as builder
+# Copies the whole project as makd needs git history:
+COPY . /project/
+WORKDIR /project/
+# Redefine arguments as env vars to be used inside build.sh script:
+ARG DIST=xenial
+ARG DMD=1.081.*
+ENV DMD=$DMD DIST=$DIST
+RUN docker/build.sh
+
+ARG DIST=xenial
+# For now plain ubuntu image is used as base for simplicity. It certainly can be
+# optimized to bare minimum but right now it is not important:
+FROM ubuntu:$DIST
```

Comment:
Do we have `deploydlang` images now available? Can we use that instead?

Reply:
Those appeared after I have updated PR last time but good idea :+1: